src/components/Reports/ReportsBox/ReportsBox.js

```diff
@@ -37,7 +37,12 @@ class ReportsBox extends React.Component {
   }
 
   componentWillReceiveProps(nextProps) {
-    this.getVisitsByPeriod(nextProps.domainName, nextProps.periodSelectedDays);
+    if (
+      nextProps.domainName !== this.props.domainName ||
+      nextProps.domainName !== this.props.domainName
```

Comment:
It seems that you accidentally are using the same property twice.

Reply:
fixed